docs/user-guide/logging/overview.md

```diff
@@ -0,0 +1,88 @@
+---
+assignees:
+- crassirostris
+- piosz
+
+---
+
+Application and systems logs is the great way to understand, what is happening inside the cluster, debug problems and monitor the activity. Most modern application have some kind of logging mechanism. Container engines are designed to support logging out of the box in some way. Easiest and the most embraced logging mechanism in the container world is to write to the standard output and standard error streams.
+
+Obviously, just that is not enough to provide a complete logging solution. For example, if container crashes, pod gets evicted or node dies, usually it's still desired to be able to access application logs. Therefore logs should have a separate storage and a separate lifecycle from nodes, pods and containers. This concept is called __cluster-level logging__ and required a separate backend to store, analyze and query logs. Kubernetes is not a logging storage solution, but it can be integrated with any of the existing ones.
+
+In the following sections firstly a basic demo of working with application logs is given. Then the node logging architecture is described in details. After that, several solutions for cluster-level logging are laid out. If you're not interested in having cluster-level logging, you might still find the description how logs are stored and handled on the node useful. For the latter section the assumption is that there exists a logging backend, inside or outside of kubernetes cluster.
+
+## Logging demo
+
+For the sake of demonstration [a following pod specification](/docs/user-guide/logging/counter-pod.yaml) is used. It has a container which writes out some text to standard output every second.
```

Comment:
"In this section, you can see an example of basic logging in Kubernetes that outputs data to the standard output stream. This demonstration uses a [pod specification](/docs/user-guide/logging/counter-pod.yaml) with a container that writes some text to standard output once per second."